doc/ansible/user-guide.md

```diff
@@ -53,7 +53,7 @@ expects this mapping file in a predefined location: `/opt/ansible/watches.yaml`
   container.  For example if your roles directory is at `/opt/ansible/roles/`
   and your role is named `busybox`, this value will be
   `/opt/ansible/roles/busybox`. This field is mutually exclusive with the
-  "playbook" field.
+  "playbook" field. If the path is relative, it is relative to the current working directory.
```

Comment:
which should be `$HOME` (`/opt/ansible`) by default, we should make sure that's mentioned

Reply:
Improved the doc desc. Please, let me know if is it ok for you now.